### ISSUE TITLE
ngninx ingress example test throwing a warning

### ISSUE THREAD

**Issue by doublethink13** [kubernetes-sigs/kind#3356]
**What would you like to be documented**:

When we `kubectl apply -f https://kind.sigs.k8s.io/examples/ingress/usage.yaml` by following https://kind.sigs.k8s.io/docs/user/ingress/#ingress-nginx, we get this warning:

```txt
Warning: path /foo(/|$)(.*) cannot be used with pathType Prefix
Warning: path /bar(/|$)(.*) cannot be used with pathType Prefix
```

According to [https://github.com/kubernetes/ingress-nginx/issues/10200#issuecomment-1673141202](https://github.com/kubernetes/ingress-nginx/issues/10200#issuecomment-1673141202), changing `pathType` to `ImplementationSpecific` does remove the warning and everything still works as expected.

**Why is this needed**:

To remove warnings.

**Comment by doublethink13** [comment]
I could have a look into this :slightly_smiling_face:

**Comment by kundan2707** [comment]
@doublethink13  are you working on this?

**Comment by stmcginnis** [comment]
Addressed by #3763 

/close
